support/tech-resources/transmissions-api-scheduling.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Scheduling Your Emails Using the Transmissions API"
+description: "Sometimes you may want your emails to be delivered at a later date and time. In this guide, we walk you through how to schedule your emails using the Transmissions API."
+---
+
+Using the Transmissions API, you can schedule your emails to be delivered up to three days in the future. All you have to do is specify a date and time in your API request, and we will not send your emails until then. Read on to learn more.
+
+## Getting Started
+
+To schedule a transmission, simply set the `options.start_time` field to the desired delivery time in your Transmissions API request. The time should be in ISO-8601 format (`YYYY-MM-DDTHH:mm:ss+-HH:mm`). We strongly encourage you to include a unique `campaign_id` in your transmission in case you need to cancel it later (more on that below). Here is an example transmission that is scheduled to be delivered on January 1, 2021 at 12:00 AM UTC:
+
+```bash
+curl -X POST \
+  https://api.sparkpost.com/api/v1/transmissions \
+  -H 'Authorization: <Your API key>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "campaign_id": "new_year",
+  "options": {
+    "start_time": "2021-01-01T00:00:00+00:00"
+  },
+  "content": {
+    "from": "example@sparkpost.com",
+    "subject": "Happy New Year!",
+    "text": "Happy New Year from the SparkPost team"
+  },
+  "recipients": [
+    {
+      "address": "recipient@sparkpost.com"
+    }
+  ]
+}'
+```
+
+## Cancelling Scheduled Transmissions
+
+If you schedule a transmission and later decide that you do not want to send it, then you can cancel it. The Transmissions API allows you to cancel scheduled transmissions by campaign ID. Simply make a `DELETE` request to the Transmissions API, providing the `campaign_id` as a query parameter. If you want to cancel a transmission scheduled by a subaccount, then you must make the request as the subaccount. You can do this by using one of the subaccount's API keys or by specifying the `X-MSYS-SUBACCOUNT` header as the master account. Here is an example of cancelling all scheduled transmissions in the `new_year` campaign:
+
+```bash
+curl -X DELETE \
+  'https://api.sparkpost.com/api/v1/transmissions?campaign_id=new_year' \
+  -H 'Authorization: <Your API key>'
+```
+
+When you cancel transmissions, the API immediately returns a 204 response, and the cancellation process happens in the background. Normally, all messages will be cancelled in ten minutes or less. In some rare circumstances, it may take longer than that. For that reason, we highly recommend that you cancel scheduled transmissions far in advance of their scheduled delivery times.
```

Comment:
nit-picky but I would switch background to "asynchronously" just because that is the wording we usually use for events and such.